service/awsconfig/v10/resource/lifecycle/delete.go

```diff
@@ -2,18 +2,8 @@ package lifecycle
 
 import (
 	"context"
-
-	"github.com/giantswarm/operatorkit/framework"
 )
 
-func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
+func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
```

Comment:
Same here. GoDoc saying delete is a no-op is useful if you're not familiar with the resource.

Reply:
Done.